src/dd_license_attribution/metadata_collector/strategies/npm_collection_strategy.py

```diff
@@ -47,6 +47,136 @@ def __init__(
             project_scope == ProjectScope.ONLY_TRANSITIVE_DEPENDENCIES
         )
 
+    def _clean_version_string(self, version: str) -> str:
+        if isinstance(version, str) and version:
+            if version.startswith(">="):
+                return version[2:]
+            elif version and version[0] in {"^", "~", ">"}:
```

Comment:
This will remove a sign that tells that this version in particular is not allowed, only newer ones are.
For current use case is ok, but wonder if we should add some comment about it so in the future we don't forget about this when adding new features.